commit ffdaadac9887aab1d6eab0b6ff6b814e822a3737
Author: Antoine R. Dumont (@ardumont) <antoine.romain.dumont@gmail.com>
Date:   Sat Oct 28 17:10:39 2023 +0200

    Add @env@ template to override in derivation

diff --git a/wee_most.py b/wee_most.py
index b5140ec..92abe16 100644
--- a/wee_most.py
+++ b/wee_most.py
@@ -17,6 +17,10 @@ import weechat
 from collections import namedtuple
 from functools import wraps
 from ssl import SSLWantReadError
+
+import sys
+sys.path.append('@env@')
+
 from websocket import (create_connection, WebSocketConnectionClosedException,
                        WebSocketTimeoutException, ABNF)
 
